### ISSUE TITLE
[Bug]: internal error: ExpectedDup: truncate 273077 error when delete from table in no-sys account

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#7220]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):56c9590273867480f531aa0ee4c539d956e09c4e
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

when run the following cases in common account, delete from table can return error

mysql> create table json_table_61(j1 json,a varchar(25),b int);
Query OK, 0 rows affected (0.10 sec)

mysql> 
mysql> insert into json_table_61 values('{"010": "beijing", "021": "shanghai"}','apple',345),('{"phonenum":"17290839029","age":"45"}','pear',0);
Query OK, 2 rows affected (0.07 sec)

mysql> 
mysql> delete from json_table_61 where b=0;
Query OK, 1 row affected (0.05 sec)

mysql> delete from json_table_61;
ERROR 20101 (HY000): internal error: ExpectedDup: truncate 273079 error
mysql>

**Comment by iamlinjunhong** [comment]
fix later

**Comment by iamlinjunhong** [comment]
fix later

**Comment by iamlinjunhong** [comment]
fix later

**Comment by iamlinjunhong** [comment]
fix later

**Comment by iamlinjunhong** [comment]
fix later

**Comment by iamlinjunhong** [comment]
This problem has not occurred in the latest version.

**Comment by aressu1985** [comment]
fixed:44f922071871a89ce50fddb69b0884a9653c8249


mysql> use test;
Database changed
mysql>  create table json_table_61(j1 json,a varchar(25),b int);
Query OK, 0 rows affected (0.06 sec)

mysql>  insert into json_table_61 values('{"010": "beijing", "021": "shanghai"}','apple',345),('{"phonenum":"17290839029","age":"45"}','pear',0);
Query OK, 2 rows affected (0.03 sec)

mysql> delete from json_table_61 where b=0;
Query OK, 1 row affected (0.03 sec)

mysql> delete from json_table_61;
Query OK, 1 row affected (0.03 sec)

mysql>
